### PR TITLE
{log-analytics} Migrate to AAZ CodeGen

### PR DESCRIPTION
**Pull request by cxznmhdcxz**
Migrate to AAZ codegen
https://github.com/Azure/aaz/pull/571

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
log-analytics

File: src/log-analytics/azext_loganalytics/commands.py [hunk 1]
```diff
@@ -5,10 +5,12 @@
...
 
 
 def load_command_table(self, _):
 
-    with self.command_group('monitor log-analytics') as g:
-        g.custom_command('query', 'execute_query', transform=transform_query_output)
+    with self.command_group('monitor log-analytics'):
+
+        from .custom import Query
+        self.command_table['monitor log-analytics query'] = Query(loader=self)
```
Comment (AllyW) [inline src/log-analytics/azext_loganalytics/commands.py:16]:
```suggestion
        self.command_table['monitor log-analytics query'] = Query(loader=self， table_transformer=action_group_list_table)
```

File: src/log-analytics/azext_loganalytics/tests/latest/test_loganalytics_commands.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ def test_query(self, resource_group):
             'az monitor log-analytics query -w {workspace_customerId} '
             '--analytics-query "Heartbeat | getschema"',
             checks=[
-                self.check("length(@)", 30),
+                self.check("length(@)", 31),
```
Comment (AllyW) [inline src/log-analytics/azext_loganalytics/tests/latest/test_loganalytics_commands.py:30]: the return schema changed?

File: src/log-analytics/azext_loganalytics/aaz/latest/monitor/log_analytics/_query.py [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+@register_command(
+    "monitor log-analytics query",
+)
+class Query(AAZCommand):
+    """Query a Log Analytics workspace
+
+    :example: Execute a simple query over past 3.5 days.
+        az monitor log-analytics query -w workspace-customId --analytics-query "AzureActivity | summarize count() by bin(TimeGenerated, 1h)" -t P3DT12H
+    """
```
Comment (AllyW) [inline src/log-analytics/azext_loganalytics/aaz/latest/monitor/log_analytics/_query.py:22]: please log examples in _help.py here.

File: src/log-analytics/azext_loganalytics/tests/latest/test_loganalytics_commands.py (hunk 2 above)
Comment (cxznmhdcxz) [reply src/log-analytics/azext_loganalytics/tests/latest/test_loganalytics_commands.py:30]: yes, there's a new property `ComputerPrivateIPs`

File: src/log-analytics/azext_loganalytics/_help.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-from knack.help_files import helps
+from knack.help_files import helps  # pylint: disable=unused-import
```
[the remark this answers is not in the document]
Comment (cxznmhdcxz) [reply src/log-analytics/azext_loganalytics/_help.py:6]: removed duplicated message

**Comment by AllyW** [comment]
please fix CI issues. @cxznmhdcxz

File: src/log-analytics/azext_loganalytics/custom.py [hunk 5]
```diff
@@ -3,12 +3,36 @@
...
+                for index, value in enumerate(row):
+                    item[column_names[index]] = str(value)
+                table_output.append(item)
+            return table_output
+
+        for table in result['tables']:
+            table_output = _transform_query_output(table)
+            tables_output.extend(table_output)
+
+        return tables_output
```
Comment (AllyW) [inline src/log-analytics/azext_loganalytics/custom.py:38]: Not necessary moved here.

**Review by AllyW — APPROVED** [review]

**Pull request merged**